appservice/src/createAppService/AppServicePlanCreateStep.ts

```diff
@@ -88,55 +88,11 @@ async function getNewPlan(context: IAppServiceWizardContext): Promise<WebSiteMan
     return plan;
 }
 
-/**
- * Has a few temporary workarounds so that the sdk allows some newer properties on the plan
- */
-function addCustomLocationProperties(plan: WebSiteManagementModels.AppServicePlan, customLocation: CustomLocation): void {
+function addCustomLocationProperties(plan: AppServicePlan, customLocation: CustomLocation): void {
     plan.perSiteScaling = true;
 
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    WebSiteManagementMappers.AppServicePlan.type.modelProperties!.kubeEnvironmentProfile = {
-        serializedName: 'properties.kubeEnvironmentProfile',
-        type: {
-            name: "Composite",
-            modelProperties: {
-                id: {
-                    serializedName: "id",
-                    type: {
-                        name: "String"
-                    }
-                }
-            }
-        }
-    };
-
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
-    (<any>plan).kubeEnvironmentProfile = { id: customLocation.kubeEnvironment.id };
-
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    WebSiteManagementMappers.AppServicePlan.type.modelProperties!.extendedLocation = {
-        serializedName: 'extendedLocation',
-        type: {
-            name: "Composite",
-            modelProperties: {
-                name: {
-                    serializedName: "name",
-                    type: {
-                        name: "String"
-                    }
-                },
-                type: {
-                    serializedName: "type",
-                    type: {
-                        name: "String"
-                    }
-                }
-            }
-        }
-    };
-
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
-    (<any>plan).extendedLocation = { name: customLocation.id, type: 'customLocation' };
+    plan.kubeEnvironmentProfile = { id: customLocation.kubeEnvironment.id };
```

Comment:
I don't really understand the mappers but based on the comments above, it was meant as a temporary workaround--and since the SDKs now support those fields I think the workaround is no longer needed.